packages/alea-frontend/pages/api/access-control/index.ts

```diff
@@ -0,0 +1,21 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { executeAndEndSet500OnError } from '../comment-utils';
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+    if (req.method == 'POST') {
+        const { description, isOpen } = req.body;
+        const id = (Math.random() * 9e6).toString(36);
```

Comment:
you can read this from the body. these would be descriptive like `ai-2-instructors` etc.